### DISCUSSION TITLE
Add support for picking from list when mentioning other user

### DISCUSSION THREAD

**Discussion by caltuntas** [d99kris/nchat#117] [Ideas]
I have been using the application mainly for our internal whatsup communication for a few days and I enjoyed a lot, thanks. It would be great to generate a popup window including all the member names of that group after putting "@" character on message window, like traditional whatsup clients do. Otherwise it's quite difficult to remember the phone numbers even names to mention someone in a chat.

**Comment by d99kris** [comment]
Hi @caltuntas - thanks for the feedback! So this feature request has come up before. Let me take a look and see if it's something that can be added without too much work. Will update here again, but might take a little while as I'm busy these few weeks.

**Reply by caltuntas** [reply]
Thanks for quick response, By the meanwhile, I'll try to sharpen my C++ skills, I would like to contribute in the future If I can.

**Comment by d99kris** [comment]
Support for mentioning other users has been added in e6e6ec6 / `v5.14.1`.

One can either type out the mentioning in text, e.g. `@Michael` (and `@[Michael Scott]` if the name has spaces), or one can use the mention / group member list dialog `alt/opt-2`.
